### PR TITLE
Feature/11153 service instance id on non bulk orders

### PR DESCRIPTION
**Pull request by bjss-andrew**
copied over the instance id for none bulk orders from develop branch to be merged into master

Story: [AB#10035](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/10035)
Story: [AB#10036](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/10036)
Story: [AB#10684](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/10684)
Task: [AB#11153](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/11153)
Task: [AB#11155](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/11155)
Task: [AB#11154](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/11154)

### CODE REVIEW

File: app/pages/summary/template.njk [hunk 1]
```diff
@@ -137,4 +137,4 @@
       </div> 
     </div>
   </div>
-{% endblock %}
+{% endblock %}
```
Comment (PhysicsJon) [inline app/pages/summary/template.njk:140]: EoF newline?

File: example.env [hunk 2]
```diff
@@ -0,0 +1,8 @@
+NODE_ENV=
+LOGGER_LEVEL=
+OIDC_BASE_URI=
+OIDC_CLIENT_SECRET=
+LOGOUT_REDIRECT_PATH=
+PUBLIC_BROWSE_BASE_URL=
+COOKIE_SECRET=
+REDIS_PASS=
```
Comment (PhysicsJon) [inline example.env:8]: EoF newline?

**Review by PhysicsJon — CHANGES_REQUESTED** [review]
Just a couple of missing EoF newlines, will need a more in depth review to ensure the correct changes have been made

File: app/pages/summary/template.njk (hunk 1 above)
Comment (bjss-andrew) [reply app/pages/summary/template.njk:140]: done

File: example.env (hunk 2 above)
Comment (bjss-andrew) [reply example.env:8]: done

**Comment by bjss-andrew** [comment]
added rule to eslint for a new line at the end
need to put in something to link template files too

**Review by bjss-chris — APPROVED** [review]

**Review by PhysicsJon — APPROVED** [review]

**Pull request merged**